code/game/turfs/simulated/floor.dm

```diff
@@ -30,6 +30,7 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 	intact = 1
 	var/broken = 0
 	var/burnt = 0
+	var/pryable = 1 //can we pry up tiles?
```

Comment:
Might have been better to make a new turf path that you cant build on/break rather than add a new var to floors